test/windows/acceptance/logstash_release_acceptance.ps1

```diff
@@ -0,0 +1,71 @@
+# Created By: Gabriel Moskovicz
+#
+# To be run on Jenkins
+#
+# Requirements to run the test:
+# 
+# - Powershell 4
+# - Windows 7 or newer
+# - Java 7 or newer
+
+$LS_CONFIG="test.conf"
+$LS_BRANCH="1.5"
```

Comment:
I don't have a preference between environment variables or flags for this kind of thing. Both will achieve the same goal which is be able to define multiple jobs or use matrix options in jenkins.

I expect environment variables to be easier to code in raw powershell than parsing command line options but I could be wrong.

What do you think @gmoskovicz ?